openbb_terminal/forecast/forecast_model.py

```diff
@@ -542,3 +542,24 @@ def corr_df(df: pd.DataFrame) -> pd.DataFrame:
         The df with the new data
     """
     return df.corr()
+
+
+@log_start_end(log=logger)
+def encode(dataset: pd.DataFrame, target_column: str = "close") -> pd.DataFrame:
+    """Converts categorical columns into integers
+
+    Parameters
+    ----------
+    dataset : pd.DataFrame
+        The dataset you wish to modify
+    target_column : str
+        The column you wish to encode
+
+    Returns
+    -------
+    pd.DataFrame:
+        Dataframe with added encoded column
+    """
+    dataset[target_column] = dataset[target_column].astype("category").cat.codes
```

Comment:
Why not the standard sklearn.preprossessing.LabelEncoder?

Reply:
Yup, that's why Colin added that description. We already talked about this 👍🏽 

We should defo be using sklearn for this.

I would still allow to have an option for this type of encoding which makes sense in the context of quantitative variables, e.g. "How hot was the water": "cold", "not much". "hot", "boiling". In here 1,2,3,4 would make sense.

Reply:
For reference: 

```python
>>> from sklearn import preprocessing
>>> le = preprocessing.LabelEncoder()
>>> le.fit(["paris", "paris", "tokyo", "amsterdam"])
LabelEncoder()
>>> list(le.classes_)
['amsterdam', 'paris', 'tokyo']
>>> le.transform(["tokyo", "tokyo", "paris"])
array([2, 2, 1]...)
```
Then we can replace the column with this newly encoded array?

Reply:
Also, for later versions: https://stackoverflow.com/questions/24458645/label-encoding-across-multiple-columns-in-scikit-learn

Reply:
> Yup, that's why Colin added that description. We already talked about this 👍🏽
> 
> We should defo be using sklearn for this.
> 
> I would still allow to have an option for this type of encoding which makes sense in the context of quantitative variables, e.g. "How hot was the water": "cold", "not much". "hot", "boiling". In here 1,2,3,4 would make sense.

Yes, but if the encoder sees cold, boiling, hot, not much, in that order, it will encode it as 1,2,3,4 even though that is not the right ordinal ordering.  What kind of strings are being encoded?

Reply:
Yup. But there are examples where preserving that relationship between variables can be important. We could allow the user to specify those :)  But prob would be better to have this as a request rather than just adding it without knowing if users would like that.